rtf

```diff
@@ -333,7 +333,7 @@ class Vagrant(Host):
     proxy = have_proxy()
     while "making sure that no vagrant up are running":
       (returncode, output) = shell("pgrep -f \"vagrant up\"", fail_exit=False)
-      if returncode != 0:
+      if returncode != 1:
```

Comment:
you mean vagrant up should return 1 in case of succes ? that doesn't look unix

Reply:
it is pgrep that returns 1 when no process are found: https://www.gsp.com/cgi-bin/man.cgi?section=1&topic=pgrep

Reply:
ok so the original ==0 was right